dbscripts/xml/upgrade.xml

```diff
@@ -151,8 +151,17 @@
 		<migration class="PKP\migration\upgrade\v3_4_0\I7592_RemoveUnusedEmailTemplates"/>
 		<migration class="PKP\migration\upgrade\v3_4_0\I7126_Galleys"/>
 		<migration class="PKP\migration\upgrade\v3_4_0\I7706_AssociateTemplatesWithMailables"/>
+<<<<<<< HEAD
+<<<<<<< HEAD
 		<migration class="APP\migration\upgrade\v3_4_0\I7190_RemoveOrphanFilters" />
 		<migration class="PKP\migration\upgrade\v3_4_0\I7190_UpdateFilters" />
+=======
+=======
+		<migration class="APP\migration\upgrade\v3_4_0\I7190_RemoveOrphanFilters" />
+		<migration class="PKP\migration\upgrade\v3_4_0\I7190_UpdateFilters" />
+>>>>>>> merge conflict fixed
+		<migration class="PKP\migration\upgrade\v3_4_0\EmailTemplatesVarcharLengthUpdate"/>
+>>>>>>> retractable/removable decision for OJS
```

Comment:
Looks like some merge conflict code snuck in here.

Reply:
fixed.